zeroclickinfo-spice/lib/DDG/Spice/Catfacts.pm

```diff
@@ -0,0 +1,21 @@
+package DDG::Spice::Catfacts;
+
+# ABSTRACT: Gives random facts about cats
+
+use DDG::Spice;
+
+spice is_cached => 0;
```

Comment:
We actually want to enable `is_cached` -- this way we know that we don't need to execute the handle funcitons again for another query that triggers this Instant Answer